README.md

```diff
@@ -80,11 +107,14 @@ https://git-scm.com/download/win
 - Install dependencies:
 `yarn install`
 
+
 ## Usage
 
 - Start the frontend:
 `yarn start`
 
+- Run the mock API:
+`yarn json-server db.json -p 8000 --watch`
```

Comment:
Does this command work for you? I think we might want `npx` instead of `yarn` here